.github/workflows/test.yml

```
@@ -0,0 +1,18 @@
+name: test
+
+on:
+  push:
+    branches: [proposed_change_not_compatible]
+  pull_request:
+    branches: [proposed_change_not_compatible]
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12
+      - run: npm i
+      - run: npm run test
```

```suggestion
      - run: npm cit
```

see https://docs.npmjs.com/cli/v7/commands/npm-install-ci-test

---

suggest, add a prettier check code style

```yaml
- run: npx prettier --list-different ./test
```

see https://prettier.io/docs/en/cli.html#--list-different

---

added